KeyVaultClientEncryption/KeyVaultClientEncryption.cs

```diff
@@ -0,0 +1,240 @@
+﻿using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage;
+using Azure.Core;
+using Azure.Security.KeyVault.Keys.Cryptography;
+using Azure.Security.KeyVault.Keys;
+using Azure.Identity;
+using System;
+using System.Text;
+using System.IO;
+
+using System.Configuration;
+using System.Diagnostics;
+
+namespace KeyClientEncryption
+{
+    class Program
+    {
+        /*
+         * SAMPLE PROGRAM THAT MIGRATES CLIENT SIDE ENCRYPTED DATA TO THREE TYPES OF SERVER SIDE ENCRYPTED DATA
+         */
+
+
+        private static void SetupForExample(BlobServiceClient blobService, string containerNameString, string fileNameString, string filePathString, ClientSideEncryptionOptions clientSideOption)
+        {
+            //Create example Container and Blob, upload blob as a txt file containing "Hello World!"
+            BlobContainerClient containerClient = blobService.CreateBlobContainer(containerNameString);
+
+            BlobClient blobClient = containerClient.GetBlobClient(fileNameString).WithClientSideEncryptionOptions(clientSideOption);

```

Comment:
We should add a comment here, cause this is important for where they should provide the `clientSideOptions` in order to get this to work for them